macros/src/test/scala/gestalt/macros/DefMacroTest.scala

```diff
@@ -79,6 +79,7 @@ class DefMacroTest extends TestSuite {
   test("def with type parameters") {
     assert(scope.is[String]("hello"))
     assert(!scope.both[String, List[Int]]("hello"))
+    scope.mapTest()
```

Comment:
Just for consistency we should look at the value, i.e.
```scala
assert(scope.mapTest() == 30)// 5*(5+1)/2*2
````
Also shouldn't it be in a different test than "def with type parameters" as it hasn't got any type params?